frontend/src/i18n/locales/en/translation.json

```diff
@@ -19,6 +19,14 @@
   "continue": "Continue",
   "ok": "OK",
   "cancel": "Cancel",
+  "internet": "Internet",
+  "local": "Local",
+  "keygen_peer_discovery_looking_for": "Looking for",
```

Comment:
"looking for" can be used in keysign as well?
Can you check vultisig-ios localization file and follow it to keep similar one?

Reply:
ok, I will you vultisig-ios localization. and update looking for.
However, it does not match the Figma design. In Figma, we have "Looking for 2 more devices," but in iOS, we only have "Looking for ...."